### PR TITLE
Metrics exporter

### PR DESCRIPTION
**Pull request by robert-nemet**
This is part of [Asana task](https://app.asana.com/0/1201167455772624/1201244386997944/f). Setting up agnostic metrics exporter for sql servers.  Needed for https://github.com/ricardo-ch/platform-charts/pull/1644

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ The public Ricardo AG Helm Charts. The code is provided as-is with no warranties
 * [imaginary](https://github.com/ricardo-ch/helm-charts/tree/main/charts/imaginary)
 * [nobl9-agent](https://github.com/ricardo-ch/helm-charts/tree/main/charts/nobl9-agent)
 * [promlens](https://github.com/ricardo-ch/helm-charts/tree/main/charts/promlens)
+* [promlens](https://github.com/ricardo-ch/helm-charts/tree/main/charts/metrics-exporter)
```
Comment (bueti) [inline README.md:13]: ~promlens~metrics-exporter

File: charts/metrics-exporter/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# metrics-exporter
+
+![Version: 1.0.0](https://img.shields.io/badge/Version-1.0.0-informational?style=flat-square) ![AppVersion: 0.5](https://img.shields.io/badge/AppVersion-0.5-informational?style=flat-square) ![Release Status](https://github.com/ricardo-ch/helm-charts/workflows/Release%20Charts/badge.svg) [![License](https://img.shields.io/github/license/ricardo-ch/helm-charts)](https://github.com/ricardo-ch/helm-charts/blob/main/LICENSE)
+
+This chart installs [the agnostic metrics exporter for sql servers](https://github.com/free/sql_exporter)
```
Comment (bueti) [inline charts/metrics-exporter/README.md:5]: This sounds like its for SQL Server (the Microsoft product). I would change it to something like this:

>This chart installs the DBMS agnostic metrics exporter based on SQL queries.

File: charts/metrics-exporter/values.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+# -- Set a service type and port
+service:
+  type: ClusterIP
+  port: 80
+
+
+# -- Set a exporter resources
+resources:
+  limits:
+    cpu: 100m
```
Comment (bueti) [inline charts/metrics-exporter/values.yaml:49]: remove the CPU limit. CPU should never be limited.

File: README.md (hunk 1 above)
Comment (robert-nemet) [reply README.md]: fixed

File: charts/metrics-exporter/README.md (hunk 2 above)
Comment (robert-nemet) [reply charts/metrics-exporter/README.md]: yes.

File: charts/metrics-exporter/values.yaml (hunk 3 above)
Comment (robert-nemet) [reply charts/metrics-exporter/values.yaml]: done.

**Review by bueti — APPROVED** [review]

**Pull request merged**